src/main/java/org/embulk/output/mailchimp/MailChimpClient.java

```diff
@@ -247,6 +249,18 @@ else if (task.getAuthMethod() == API_KEY && task.getApikey().isPresent()) {
         }
     }
 
+    private void findList(final PluginTask task)
+    {
+        String endpoint = MessageFormat.format(mailchimpEndpoint + "/lists/{0}",
+                                               task.getListId());
+        try {
+            client.sendRequest(endpoint, HttpMethod.GET, task);
+        }
+        catch (HttpResponseException hre) {
+            throw new ConfigException("The `list id` could not be found.");
```

Comment:
The `endpoint` or actual `list id` within the ConfigException message may help with trouble shooting.

Reply:
Actually, we did log before throw the exception

https://github.com/treasure-data/embulk-output-mailchimp/pull/38/files#diff-0e2f506a17351bcc32943da8e0080b28R107